### DISCUSSION TITLE
Query to select part of a node

### DISCUSSION THREAD

**Discussion by schneedorfer** [nvim-treesitter/nvim-treesitter#5060] [Q&A]
So I am trying to select the quotes from a string and color them differently than the string content itself. However the whole string including the quotes is marked as a \@string highlight. Is this somehow possible?

**Comment by amaanq** [comment]
Depends on the grammar of the language, of course

**Comment by clason** [comment]
You might be able to fake something with `#offset!` and multiple queries, though.

**Reply by schneedorfer** [reply]
hmmm, can I somehow create a different capture with the offset? Or would I have to do this in code, since the docs say `This will generate a new range object for the captured node as metadata[capture_id].range`. I dont really understand what to do with that..
